Islandora/islandora_badges
# Change oaDOI to Unpaywall

**JIRA Ticket**: (https://jira.duraspace.org/browse/ISLANDORA-2224)

# What does this Pull Request do?

Removes the oaDOI badge, creates an Unpaywall badge to replace it.

# What's new?
New name, new API endpoint, adjustment to data processing, new admin form fields (email, API endpoint)

# How should this be tested?

- Have an object with a DOI that is recognized by Unpaywall (easy if you've got oaDOI installed)
- Check out this branch, enable module, enable block
- See if block appears on that object
- Make sure block doesn't do anything else that's weird

# Additional Notes:
This may need an update hook to uninstall oaDOI...? I don't know how that works.

# Interested parties
@rosiel  @Islandora/7-x-1-x-committers
